CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Removed
 - Stop using Prophecy (#676)
+- Drop ddev configuration in favor of runTests.sh (#1063)
```

Comment:
This needs to go into the current milestone, and it should be on top of the "Removed" section.

Also:
```suggestion
- Drop DDEV configuration in favor of `runTests.sh` (#1063)
```